util/chunk/chunk.go

```diff
@@ -706,3 +706,22 @@ func (c *Chunk) ToString(ft []*types.FieldType) string {
 	}
 	return string(buf)
 }
+
+// AppendRows appends multiple rows to the chunk.
```

Comment:
Maybe it's better to make the comment of these two functions `AppendRows` and `AppendPartialRows` more clearly.